config/attributes/character.yml

```diff
@@ -0,0 +1,115 @@
+:overview:
+  :label: Overview
+  :icon: info
+  :attributes:
+    - :name: name
+      :label: Name
```

Comment:
That's true, but since these values are no longer part of the ActiveModel object, I'm not sure how to leverage that machinery.